pkgs/top-level/all-packages.nix

```diff
@@ -9649,8 +9649,6 @@ in
 
   epoxy = callPackage ../development/libraries/epoxy {};
 
-  esdl = callPackage ../development/libraries/esdl { };
```

Comment:
From GitHub of esdl project:
Esdl is a legacy library for accessing SDL and OpenGL from Erlang. (New applications is better off using wx in Erlang/OTP R13B.)
In NixPkgs it was used only by Wings3d, but with my update, new Wings3d does not use it anymore.

Reply:
@tex I see. Because other people might still be referring to that library in their code, I think it's a good idea to not just remove it, but replace it with a `throw` and an appropriate error message.

Reply:
I do not know where to put the throw and what to show as error.

If anyone is refereing to this package then he finds that it was removed and knows he needs to get it from older version. What is generally strategy about removing packages from NixPkgs?

The Esdl has last been touched ~6 years ago... only supports prehistoric version of SDL and Erlang.

Reply:
Oh I guess it's fine then. The warning would have to be at this very line, `esdl = throw "esdl is a legacy library, use wx instead"`